blog-service/2024-05-01-apps.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Expanding Cloud Infrastructure Security for AWS Capabilities - Preview (Apps)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - apps
+  - app catalog
+  - cloud infrastructure security
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce increased visibility into your AWS Cloud environment with the following new features:
+* **Out-of-the-box security policy checks**. Cloud Infrastructure Security is now configured by default to use the out-of-the box policy checks. You can now choose to leverage the out-of-the-box policy checks instead of, or in conjunction with, the policy checks provided by AWS Security Hub. 
```

Comment:
To distinguish it from the AWS-branded products mentioned here

```suggestion
* **Out-of-the-box security policy checks**. Sumo Logic Cloud Infrastructure Security is now configured by default to use the out-of-the box policy checks. You can now choose to leverage the out-of-the-box policy checks instead of, or in conjunction with, the policy checks provided by AWS Security Hub. 
```